portfolio/src/main/webapp/src/com.js

```diff
@@ -115,25 +117,37 @@ function deleteAllCommentsFromServer() {
 }
 
 /**
- * Creates a comment card with the given comment text
+ * Creates a comment card with the given comment poster, comment text
  * and sentiment score if it is provided.
  *
  * @param {string} commentText
+ * @param {any} commentPoster
```

Comment:
Here and below, why are these annotated as `any`?

Reply:
Will change the `Promise<any>` annotations to `Promise<Object>` annotations, as it is now well-defined what servlets return.

As for the `any` annotations on `createCommentCard`, is there a specific way of annotating a parameter's type when it could potentially be undefined? Or do I just use `{string}`?